thirdparty/build-thirdparty.sh

```diff
@@ -1345,8 +1345,13 @@ build_hdfs3() {
     cd "${BUILD_DIR}"
     rm -rf ./*
 
+    if [[ "$(uname -m)" == "aarch64" ]]; then
+        SSE_OPTION='-DENABLE_SSE=OFF'
+    else
+        SSE_OPTION='-DENABLE_SSE=ON'
+    fi
```

Comment:
```suggestion
    if [[ "$(uname -m)" != 'x86_64' ]]; then
        SSE_OPTION='ON'
    else
        SSE_OPTION='OFF'
    fi
```

It is better to use `x86_64` to compare to the result due to the result of `uname -m` is `arm64` on macOS M1.

BTW, it seems that the compiler just complains the unrecognized option and continues. The error doesn't interrupt the compilation.

Reply:
I found this when I try to compile on the 'TencentOS Server 3.1 for ARM64 (TK4)' system, it does interrupt the compilation.

Reply:
> I found this when I try to compile on the 'TencentOS Server 3.1 for ARM64 (TK4)' system, it does interrupt the compilation.

Ok.